src/app/modules/main/wallet_section/module.nim

```diff
@@ -228,13 +251,18 @@ method overviewModuleDidLoad*(self: Module) =
 method sendModuleDidLoad*(self: Module) =
   self.checkIfModuleDidLoad()
 
+method networksModuleDidLoad*(self: Module) =
+  self.checkIfModuleDidLoad()
+
 method destroyAddAccountPopup*(self: Module, switchToAccWithAddress: string = "") =
-  if not self.addAccountModule.isNil:
-    if switchToAccWithAddress.len > 0:
-      self.switchAccountByAddress(switchToAccWithAddress)
-    self.view.emitDestroyAddAccountPopup()
-    self.addAccountModule.delete
-    self.addAccountModule = nil
+  if self.addAccountModule.isNil:
+    return
+  if switchToAccWithAddress.len > 0:
+    self.setFilterAddress(switchToAccWithAddress)
```

Comment:
@alaibe with the approach you've implemented here we don't need lines `260` and `261`, as well you're free to remove `switchToAccWithAddress` from the method declaration, cause now `setFilterAddress` will be definitely called when a new account is added as a result of running add account popup.

Reply:
Perfect, will do